File: count.R
print("R language")

start_time <- Sys.time()
i <- 0
while (i < 1000000000) {
  i <- i + 1
}
end_time <- Sys.time()
print(paste("time:", as.numeric(difftime(end_time, start_time, units = "secs")), "seconds"))

File: count.java
public class count {
    public static void main(String[] args) {
        System.out.println("Java language");
        int i = 0;
        long startTime = System.currentTimeMillis();
        while (i < 1_000_000_000) {
            i++;
        }
        long endTime = System.currentTimeMillis();
        System.out.println("time:" + (endTime - startTime) + " ms");
    }
}
